Service/NavitiaService.php

```diff
@@ -68,7 +68,7 @@ public function processConfiguration($config)
      */
     public function process($query, $format = null, $timeout = 6000, $pagination = true)
     {
-        $this->timeout = $timeout;
+        $this->timeout = max($timeout, $this->config->getTimeout());
```

Comment:
`max` is perhaps not the best option.
If you set less than 6000, your config won't be used.